### DISCUSSION TITLE
StringTokenizer

### DISCUSSION THREAD

**Discussion by mssanamateen** [mssanamateen/OOP-using-Java#40] [Q&A] [answered]
Why do we use it and what is the third constructor for creating it's object.. Write the syntax and mention your point of view.

**Comment by MYethishwar** [comment, marked as answer]
String Tokenizer is used to break a string into tokens(parts) based on a delimiter(separator).

Syntax for 3rd Constructor: 
StringTokenizer(String str, String delim, boolean returnDelims)

**Comment by Maithri1234** [comment]
StringTokenizer class in Java is used to break a string into tokens(parts)
SYNTAX FOR THIRD CONSTRUCTOR:
StringTokenizer(String str, String delim, boolean return value)

**Comment by RAHUL-KOPPULA** [comment]
String Tokenizer is used to break a string into tokens(parts) based on a delimiter(separator).

Syntax for 3rd Constructor:
StringTokenizer(String str, String delim, boolean returnDelims)

**Comment by abhi15102004** [comment]
We use StringTokenizer in Java to break a string into tokens based on a specified delimiter. It provides a simple way to tokenize a string and iterate over the tokens.

The third constructor for creating a StringTokenizer object is:
public StringTokenizer(String str, String delim, boolean returnDelims)
